raiwidgets/tests/test_responsibleai_dashboard_input.py

```diff
@@ -1,24 +1,222 @@
 # Copyright (c) Microsoft Corporation
 # Licensed under the MIT License.
 
-from unittest.mock import patch
-
+from raiwidgets.interfaces import WidgetRequestResponseConstants
 from raiwidgets.responsibleai_dashboard_input import \
     ResponsibleAIDashboardInput
 
 
-class TestResponsibleAIDashboardInput:
-    def test_model_analysis_adult(
+class TestResponsibleAIDashboardInputClassification:
+    def test_model_analysis_adult_on_predict_success(
```

Comment:
nit: change all test names to use rai_insights or responsibleai since model_analysis isn't a term we want to use anymore

Reply:
Done. Change to rai_dashboard_input instead of model_analysis.